### DISCUSSION TITLE
Where are the rts-sources for the GNAT FSF compiler?

### DISCUSSION THREAD

**Discussion by JCGobbi** [AdaCore/bb-runtimes#65] [Ideas]
I have ported the Ada runtime for the STM32F334, STM32G474 and STM32H743 microprocessors and compiled them with the GNAT Community edition 2021, that has the rts-sources within it (at arm-eabi/include/rts-sources). The GNAT FSF compiler doesn't have that sources, so I copied then to bb-runtimes/src/rts-sources and tried to build that sources with the option --rts-src-descriptor="src/rts-sources.json", so the build_rts.py may use them. But the GNAT FSF doesn't compile that sources, some files need changes.
But you have compiled that sources for the supported microprocessors that come with GNAT FSF compiler. How can I get that modified rts-sources to update my runtimes with the GNAT FSF compiler? Perhaps that sources may be part of the bb-runtimes inside bb-runtimes/src.

**Comment by Fabien-Chouteau** [comment]
Hello @JCGobbi 

See my answer here: https://github.com/AdaCore/bb-runtimes/issues/60#issuecomment-1444026566

**Reply by JCGobbi** [reply]
Thank you, it works fine.
